mscore/workspace.cpp

```diff
@@ -1003,15 +1007,14 @@ QList<Workspace*>& Workspace::workspaces()
                         
                         if (!p) {
                               p = new Workspace;
-                              p->setPath(s + "/" + entry);
-                              p->setName(name);
-                              
-                              if (translate)
-                                    p->setTranslatableName(name);
-                              
-                              p->setReadOnly(!fi.isWritable());
-                              _workspaces.append(p);
-                              }
```

Comment:
This brace needs to stay. Or the opening one needs to go

Reply:
Ah, merging empty strings is a nightmare! Thanks for catching it :)